web/src/components/CollectionItemCard/ReadOnlineLink.tsx

```diff
@@ -17,13 +19,23 @@ const ReadOnlineLink: React.FC<{ links: OpdsLink[]; title: string }> = ({
 
   if (!readOnlineLink) return null;
 
+  const trackReadOnlineClick = () => {
+    trackEvent({
+      "event":  "digital_read_online",
+      "item_title": title,
+      "item_author": author,
```

Comment:
I wonder if it would makes more sense to enforce this to a specific type (make `item_author` either an array or a string)?  Might be easier to process in the dashboard consider GA might treat the two datatypes differently, but please ignore if you think this is not necessary!

Reply:
Miguel said that both values, either array or string, are valid for item_author.